src/core/server/saved_objects/migrations/core/document_migrator.test.ts

```diff
@@ -20,14 +20,25 @@
 import _ from 'lodash';
 import { RawSavedObjectDoc } from '../../serialization';
 import { DocumentMigrator } from './document_migrator';
+import { Logger } from 'src/core/server/logging';
+
+const mockLogger: jest.Mocked<Logger> = {
```

Comment:
there is already Logger mock https://github.com/restrry/kibana/blob/6e1a527d4a8e4bf9beb8e0e8dd0e7eb966f00843/src/core/server/logging/logging_service.mock.ts